doc/development/04_wagons.md

```diff
@@ -253,11 +253,14 @@ If wished, the attributes can be included in the CSV-File that is generated when
 
 #### Make attributes searchable
 
-The new attributes must be indexed in `app/indices/person_index.rb` where all indexes for Sphinx (the search tool that is used by hitobito) are defined.
-
-    ThinkingSphinx::Index.define_partial :person do
-      indexes title
-    end
+To make new attributes searchable you have to include them in the searchable attributes
+of the corresponding model:
+
+    SEARCHABLE_ATTRS = [
+    :first_name, :last_name, :company_name, :nickname, :email, :address, :zip_code, :town,
+    :country, :birthday, :additional_information, {phone_numbers: [:number],
+                                                   social_accounts: [:name], additional_emails: [:email]}
+]
```

Comment:
Also document the search_strategies domain classes and add some documentation that PgSesarchable has to be included in the corresponding model